### PR TITLE
Issue #28: Fix bug in heuristic

### LINKED ISSUE #28: Fix Heuristic implementation

### PR DESCRIPTION
**Pull request by David6114**
Fixed the faulty logic in the heuristic. 
Resolves #28

### CODE REVIEW

File: game_of_agents.py [hunk 1]
```diff
@@ -50,20 +52,25 @@ def inrow_heuristic(env, player, weight=0):
...
+                        score += score_func(counter_p + counter_n, weight)
                     else:
-                        score += (counter_p)**5 + (counter_n)**5
+                        score += score_func(counter_p, weight) + \
+                            score_func(counter_n, weight)
+    return score
+
+
+def score_func(count, weight):
+    score = count**POWER
```
Comment (levzlotnik) [inline game_of_agents.py:63]: This function doesn't return anything, did you test your code by running the tournament script?

Comment (levzlotnik) [reply game_of_agents.py:63]: Also, weight is not used in `score_func` - why are you passing it then?

**Review by levzlotnik — CHANGES_REQUESTED** [review]

File: game_of_agents.py (hunk 1 above)
Comment (David6114) [reply game_of_agents.py:63]: Did you see the all code of this function before you requested changes?

Comment (levzlotnik) [reply game_of_agents.py:63]: Oh sorry, I didn't notice the lines below belonged to this function.

**Review by levzlotnik — APPROVED** [review]

**Review by nabaco — APPROVED** [review]

**Pull request merged**